packages/styles/scss/components/tabs-extended/_tabs-extended.scss

```diff
@@ -48,6 +52,7 @@
             @include type-style('productive-heading-01');
 
             color: $text-primary;
+            line-height: 1.375rem;
```

Comment:
The font tokens should already set the `line-height`, this is something we want to overwrite?

Reply:
@annawen1, the vertical text alignment was a bit off in the selected tab. It seems the line-height set by `productive-heading-01` is different than the other tabs. This is how it looks like without the overwrite:

![image](https://user-images.githubusercontent.com/4054756/127515467-db48fd82-7749-42dc-87af-10b9b98a4fba.png)

It's 1px highter.

Reply:
@NSDrowned thanks for fixing the text alignment! 

I think we could remove the line height override by switching the tab items to use $body-short-01. Carbon has some funky pairings with the heading and body styles that I didn't think about when I wrote the issue initially. 

Apologies for the back and forth, would we be able to remove the line height override in the heading style and switch from $body-long-01 to $body-short-01 ?

Reply:
@oliviaflory I switched to `body-short-01` but still have the same problem. The line-height problem seems to lie in `productive-heading-01`.

Reply:
Talked with Olivia over slack and we're going with the override for now 👍